tools/api_check/lib/apicheck.dart

```diff
@@ -52,6 +53,16 @@ List<String> getDartClassFields({
   return fields;
 }
 
+NodeList<ClassMember>? _getMembers(String className, CompilationUnitMember unitMember) {
```

Comment:
suggestion for simplification: type this as non-nullable and make it return an empty list instead of null. Then you can also remove the null check on line 39.

Reply:
TIL NodeList can be a []. Hold on!

Reply:
Oh, instead of NodeList you could also just use List since `NodeList` implements `List`. You may actually have to do that to do the empty list thing.

Reply:
It turns out it couldn't... lol